src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/germline/CoverageModellerGermlineSparkToggleIntegrationTest.java

```diff
@@ -0,0 +1,398 @@
+        runLearningAndCallingTest();
+    }
+
+    @Test(dependsOnMethods = "runLearningAndCallingTestSpark")
+    public void runCaseSampleCallingTestOnLearnedModelParamsSpark() {
+        runCaseSampleCallingTestOnLearnedModelParams();
+    }
+
+    @Test
+    public void runCaseSampleCallingTestOnExactModelParamsSpark() {
+        runCaseSampleCallingTestOnExactModelParams();
+    }
+
+    /* Shame on me for using {@link ReadCountCollection} to store copy numbers! */
+    private void assertInferredCopyNumberConcordance(@Nonnull final File posteriorsOutputPath,
+                                                     @Nonnull final File truthCopyNumberFile,
+                                                     @Nonnull final List<Target> targets,
+                                                     @Nonnull final SexGenotypeDataCollection sexGenotypeDataCollection) {
+        final ReadCountCollection truthCopyNumberCollection = loadTruthCopyNumberTable(truthCopyNumberFile,
+                targets);
+
+        final RealMatrix inferredCopyNumberMatrix = Nd4jApacheAdapterUtils.convertINDArrayToApacheMatrix(
+                Nd4jIOUtils.readNDArrayFromTextFile(new File(posteriorsOutputPath,
+                        CoverageModelGlobalConstants.COPY_RATIO_VITERBI_FILENAME)).transpose());
+        final ReadCountCollection inferredCopyNumberCollection = new ReadCountCollection(targets,
+                truthCopyNumberCollection.columnNames(), inferredCopyNumberMatrix);
+
+        final int numSamples = inferredCopyNumberCollection.columnNames().size();
+        final List<String> sampleSexGenotypes = truthCopyNumberCollection.columnNames().stream()
+                .map(sampleName -> sexGenotypeDataCollection.getSampleSexGenotypeData(sampleName).getSexGenotype())
+                .collect(Collectors.toList());
+
+        final List<SampleCopyNumberConcordanceSummary> sampleConcordanceSummaryList = IntStream.range(0, numSamples)
+                .mapToObj(si -> calculateSampleCopyNumberConcordance(truthCopyNumberCollection,
+                        inferredCopyNumberCollection, si, sampleSexGenotypes.get(si)))
+                .collect(Collectors.toList());
+
+        /* calculate average concordances */
+        final double averageRefConcordance = IntStream.range(0, numSamples)
+                .mapToDouble(si -> sampleConcordanceSummaryList.get(si).refConcordance)
+                .sum() / numSamples;
+        final double averageAltConcordance = IntStream.range(0, numSamples)
+                .mapToDouble(si -> sampleConcordanceSummaryList.get(si).altConcordance)
+                .sum() / numSamples;
+        final double[] averageAbsConcordance = IntStream.range(0, numSamples)
+                .mapToObj(si -> sampleConcordanceSummaryList.get(si).absConcordance)
+                .reduce(new double[MAX_COPY_NUMBER + 1], (a, b) -> {
+                    final double[] res = new double[MAX_COPY_NUMBER + 1];
```

Comment:
I actually updated/improved the concordance part of the test significantly in the post-PR commit. This is not relevant anymore.